src/components/instance-creation-page/GeneralInfoForm.jsx

```diff
@@ -81,7 +81,7 @@ class GeneralInfoForm extends React.Component {
                 value: instance.name,
                 component: TextField,
                 props: {
-                    floatingLabelText: i18n.t("Server name (*)"),
```

Comment:
Is it worth to create some `function optional(s) { return `${s} (*)`; } in some utils file to abstract it? Not sure.

Reply:
Both options are fine